test/bundle.js

```diff
@@ -35,30 +35,22 @@ mocha.describe('watching', function () {
     fs.writeFileSync(file2.name, makeRequire(file3.name));
     fs.writeFileSync(file3.name, makeRequire(file4.name));
 
-    bundle = galv.bundle({
-      paths: [
-        file1.name,
-        file2.name
-      ],
-      options: {
-        common: file1.name
-      }
-    });
```

Comment:
I don't see any code in treshugart/ubercod that supports passing positional parameters as named parameters in one object.

This call was failing with
```text
   1) watching "before each" hook:
     TypeError: glob pattern string required
      at new Minimatch (/Users/nvenegas/Code/galvatron/node_modules/minimatch/minimatch.js:108:11)
      at setopts (/Users/nvenegas/Code/galvatron/node_modules/glob/common.js:123:20)
      at new GlobSync (/Users/nvenegas/Code/galvatron/node_modules/glob/sync.js:38:3)
      at Function.globSync (/Users/nvenegas/Code/galvatron/node_modules/glob/sync.js:24:10)
      at /Users/nvenegas/Code/galvatron/src/glob.js:14:10
      at Array.forEach (native)
      at module.exports (/Users/nvenegas/Code/galvatron/src/glob.js:13:12)
      at Object.Bundle (/Users/nvenegas/Code/galvatron/src/bundle.js:26:16)
```

since `bundle()` was receiving the object with keys `paths` and `options` as the `paths` argument.